pkg/utils/accelerators/tpu.go

```diff
@@ -79,6 +82,84 @@ func getContainerRequestingTPUs(spec *corev1.PodSpec) *corev1.Container {
 	return nil
 }
 
+func AddTPUVariablesSubGroup(pod *corev1.Pod, size int) error {
+	container := getContainerRequestingTPUs(&pod.Spec)
+	if container == nil {
+		return nil
+	}
+
+	for _, env := range container.Env {
+		if env.Name == TpuWorkerHostNames || env.Name == TpuWorkerId {
+			return nil
+		}
+	}
+
+	leaderName := pod.Name
+	subGroupSize, err := strconv.Atoi(pod.Annotations[leaderworkerset.SubGroupSizeAnnotationKey])
+	if err != nil {
+		return err
+	}
+
+	subGroupIndex, err := strconv.Atoi(pod.Labels[leaderworkerset.SubGroupIndexLabelKey])
+	if err != nil {
+		return err
+	}
+
+	workerIndex, err := strconv.Atoi(pod.Labels[leaderworkerset.WorkerIndexLabelKey])
+	if err != nil {
+		return err
+	}
+	tpuWorkerId := (workerIndex) % subGroupSize
+
+	if pod.Annotations[LeaderRequestsTPUsAnnotationKey] != "true" {
+		tpuWorkerId = (workerIndex - 1) % subGroupSize
+	}
+
+	start := subGroupSize*subGroupIndex + 1
```

Comment:
```suggestion
	start := subGroupSize * subGroupIndex + 1
```

Reply:
did you forgot to update this one? if we have space in the next line for '*', then we need to be consistent